### PR TITLE
chore: capitalize ScanMetadata consistently

### PR DESCRIPTION
**Pull request by pownkel**
#### Description of changes

some props have a property called scanMetaData, while some capitalize it as scanMetadata. This will standardize all variables with this name to scanMetadata.

#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [n/a] Added/updated relevant unit test(s) (and ran `yarn test`)
- [n/a] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [n/a] (UI changes only) Added screenshots/GIFs to description above
- [n/a] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/common/types/store-data/scan-meta-data.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { ToolData } from 'common/types/store-data/unified-data-interface';
 
 import { TargetAppData } from 'common/types/store-data/unified-data-interface';
 
-export type ScanMetaData = {
+export type ScanMetadata = {
```
Comment (dbjorge) [inline src/common/types/store-data/scan-meta-data.ts:7]: Should the file be updated to `scan-metadata.ts` for consistency with the new type name? (fine to leave this for a separate PR to split up review)

Comment (pownkel) [reply src/common/types/store-data/scan-meta-data.ts:7]: Good point, I'll address it in a seperate pr
